master/buildbot/data/forceschedulers.py

```diff
@@ -27,7 +27,8 @@ def forceScheduler2Data(sched):
                name=unicode(sched.name),
                button_name=unicode(sched.buttonName),
                label=unicode(sched.label),
-               builder_names=map(unicode, sched.builderNames))
+               builder_names=map(unicode, sched.builderNames),
```

Comment:
while you are at it, do you think you could convert this `map` to a list comprehension?  (that'd be a bit more readable)